.devcontainer/devcontainer.json

```diff
@@ -17,17 +17,6 @@
 			"settings": {
 				"python.defaultInterpreterPath": "/workspace/cs-env/bin/python",
 				"python.editor.tabSize": 2,
-				"python.linting.enabled": true,
-				"python.linting.pylintArgs": [
-					"--indent-string=\"  \""
-				],
-				"python.linting.banditEnabled": true,
-				"python.linting.mypyEnabled": true,
-				"python.linting.pylintEnabled": true,
-				"python.formatting.yapfPath": "/usr/local/py-utils/bin/yapf",
-				"python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
-				"python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
-				"python.linting.pylintPath": "/workspace/cs-env/bin/pylint",
```

Comment:
We need to force pylint to use two spaces instead of two. The [extension](https://marketplace.visualstudio.com/items?itemName=ms-python.pylint) allows us to configure the setting.

```
"pylint.args": ["--indent-string=\"  \""]
```

Reply:
Similarly, can you add a config for mypy to match what is used in the package.json

```
"mypy-type-checker.args": ["--ignore-missing-imports --exclude cs-env/ --exclude gen/ --exclude appengine_config.py --disable-error-code \"annotation-unchecked\""]
```